### PR TITLE
core: fix retry for transfer image stopImageTransferSession failure

### PR DESCRIPTION
**Pull request by bennyz**
Currently when a transfer will fail stopImageTransferSession as part of
handleFinalizingSuccess it will finish successfully, leaving garbage
behind and the image will remain locked.

This patch is a partial revert of c54d500, enabling cleanup retries if
stopping the transfer session fails.

Bug-Url: https://bugzilla.redhat.com/2057445

### CODE REVIEW

**Comment by sandrobonazzola** [comment]
waiting on BZ approval for backport to 4.4.10-2

**Review by ahadas — APPROVED** [review]

**Comment by sandrobonazzola** [comment]
acks granted, merging

**Pull request merged**
